gravitee-apim-portal-webui/src/app/pages/catalog/filtered-catalog/filtered-catalog.component.ts

```diff
@@ -282,21 +287,8 @@ export class FilteredCatalogComponent implements OnInit {
   }
 
   onSelectCategory({ target }) {
-    if (target.value !== '') {
-      const queryParams = { category: target.value };
-      queryParams[SearchQueryParam.PAGE] = 1;
-      this.router.navigate([], {
-        relativeTo: this.activatedRoute,
-        queryParams,
-        queryParamsHandling: 'merge',
-        fragment: this.fragments.filter,
-      });
-    } else {
-      this.router.navigate([], {
-        relativeTo: this.activatedRoute,
-        fragment: this.fragments.filter,
-      });
-    }
```

Comment:
it didn't seem to work but I think the idea of routing was to have the filter in the url, isn't it possible to keep it?

Reply:
My bad, I didn't get the real purpose of this part of the code. I just reverted my changes on this method and it's still ok for my fix. I will just rework on my unit tests

Reply:
Done

Reply:
Done